addon/index.js

```diff
@@ -124,8 +124,6 @@ class EmberRouterScroll extends EmberRouter {
     if (callbackRequestId) {
       window.cancelAnimationFrame(callbackRequestId);
     }
-
-    super.destroy(...arguments);
```

Comment:
Do we still want `super` here?

Reply:
@snewcomer good point. Reverted, just in case